src/micro_http/src/connection.rs

```diff
@@ -406,6 +406,20 @@ mod tests {
     use super::*;
     use common::{Method, Version};
     use std::os::unix::net::UnixStream;
+    use MediaType;
+
+    impl Headers {
```

Comment:
I don't think this change is needed. We are trying to test the connection here, so the accept header is not necessarily needed. We should be keeping this file as it was before.